tracing-subscriber/src/filter/env/directive.rs

```diff
@@ -178,13 +178,13 @@ impl FromStr for Directive {
         lazy_static! {
             static ref DIRECTIVE_RE: Regex = Regex::new(
                 r"(?x)
-                ^(?P<global_level>trace|TRACE|debug|DEBUG|info|INFO|warn|WARN|error|ERROR|off|OFF|[0-5])$ |
+                ^(?P<global_level>(?i:trace|debug|info|warn|error|off|[0-5]))$ |
```

Comment:
maybe nice to add a regex comment noting that the level is case-insensitive? not a blocker.

Reply:
Same with this one -- it's in my most recent push.